### PR TITLE
Add ColdFront

### LINKED ISSUE #29: Add ColdFront 2018
https://2018.coldfront.co/

💪

### PR DESCRIPTION
**Pull request by auchenberg**
Fixes #29

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,6 +171,7 @@ Additions/corrections are welcome! Please check out [Contributing Guidelines](CO
 | [AngularConnect](http://angularconnect.com/) | London, United Kingdom | November 7 - 8, 2017 |  May 31, 2017 | Yes | [Yes](http://angularconnect.com/2017/code-of-conduct/) |
 | [DeveloperWeek Austin JavaScript Conference](http://www.developerweek.com/Austin/) | Austin, Texas | November 7 - 9, 2017 |  September 13, 2017 | No | [Yes](http://confcodeofconduct.com/) |
 | [ffconf](https://ffconf.org) | Brighton, United Kingdom | November 8 - 9, 2018 |  Unknown | Yes | [Yes](https://ffconf.org/code-of-conduct) |
+| [ColdFront](https://2018.coldfront.co) | Copenhagen, Denmark | November 13 - 14, 2017 |  June 18, 2017 | Yes | [Yes](http://confcodeofconduct.com/) |
```
Comment (benmvp) [inline README.md]: Can you update these dates to 2018? Also it looks like the call for recommendations ends May 1st 2018? https://2018.coldfront.co/call-for-recommendations

**Review by benmvp — COMMENTED** [review]
Thanks for this! Left you a quick comment.

**Comment by auchenberg** [comment]
Updated

**Comment by benmvp** [comment]
Thanks again!

**Pull request merged**